pytest.ini

```diff
@@ -1,2 +1,8 @@
 [pytest]
-norecursedirs = .* dist CVS _darcs {arch} *.egg
\ No newline at end of file
+DJANGO_SETTINGS_MODULE = autoreduce_webapp.test_settings
+norecursedirs = .* dist CVS _darcs {arch} *.egg
+# We have to pretend that the root of the project also exists in
+# WebApp/autoreduce_webapp since trying to import stuff through it
+# will break Django. (Really we should split out everything in
+# WebApp/autoreduce_webapp into its own repo in the future)
+python_paths = WebApp/autoreduce_webapp
```

Comment:
Could defining python_paths in top level pytest.ini have a side effect somewhere else?

Reply:
I don't think anything reads these config files apart from the tools they're intended for - this should be OK